packages/flame/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
  - Add clear function to BaseGame to allow the removal of all components
  - Moving tutorials to the Flame main repository
  - Exporting Images and AssetsCache
+ - Remove the SpriteAnimationComponent when the animation is really done, not when it is on the last frame (this is a breaking change)
```

Comment:
I know our PR states that you have to indicate a breaking change, but as this is for the v1 branch. Which is not officially released yet I dont think we need to indicate these breaking changes.

```suggestion
 - *Remove the SpriteAnimationComponent when the animation is really done, not when it is on the last frame
```

Reply:
I think so.